src/frontend/js/media-management/component/edit-sidebar.tsx

```diff
@@ -277,6 +278,22 @@ export default function EditSidebar({
                       {mediaTranslations.btn_save_file}
                     </button>
                   )}
+
+                  <label
+                    for="replace-file-input"
+                    // title={mediaTranslations.btn_replace_file} 
```

Comment:
```suggestion
                    title={mediaTranslations.btn_replace_file} 
```
I think we can very well use this title here.